docs/tutorial/debugging-main-process-node-inspector.md

```diff
@@ -0,0 +1,132 @@
+# Debugging the Main Process in node-inspector
+
+[`node-inspector`][node-inspector] provides a familiar DevTools GUI that can
+be used in Chrome to debug Electron's main process, however, because
+`node-inspector` relies on some native Node modules they must be rebuilt to
+target the version of Electron you wish to debug. You can either rebuild
+the `node-inspector` dependencies yourself, or let
+[`electron-inspector`][electron-inspector] do it for you, both approaches are
+covered in this document.
+
+**Note**: At the time of writing the latest release of `node-inspector`
+(0.12.8) can't be rebuilt to target Electron 1.3.0 or later without patching
+one of its dependencies. If you use `electron-inspector` it will take care of
+this for you.
+
+
+## Use `electron-inspector` for Debugging
+
+### 1. Install the [node-gyp required tools][node-gyp-required-tools]
+
+### 2. Install [`electron-rebuild`][electron-rebuild], if you haven't done so already.
+
+```shell
+npm install electron-rebuild --save-dev
+```
+
+### 3. Install [`electron-inspector`][electron-inspector]
+
+```shell
+npm install electron-inspector --save-dev
+```
+
+### 4. Start Electron
+
+Launch Electron with the `--debug` switch:
+
+```shell
+electron --debug=5858 your/app
+```
+
+or, to pause execution on the first line of JavaScript:
+
+```shell
+electron --debug-brk=5858 your/app
+```
+
+### 5. Start electron-inspector
+
+On macOS / Linux:
+
+```shell
+node_modules/.bin/electron-inspector
+```
+
+On Windows:
+
+```shell
+node_modules\\.bin\\electron-inspector
+```
+
+`electron-inspector` will need to rebuild `node-inspector` dependencies on the
+first run, and any time you change your Electron version. The rebuild process
+may require an internet connection to download Node headers and libs, and may
+take a few minutes.
+
+### 6. Load the debugger UI
+
+Open http://127.0.0.1:8080/debug?ws=127.0.0.1:8080&port=5858 in the Chrome
+browser. You may have to click pause if starting with `--debug-brk` to force
+the UI to update.
+
+[electron-inspector]: https://github.com/enlight/electron-inspector
+[electron-rebuild]: https://github.com/electron/electron-rebuild
```

Comment:
What do you think about moving these to the bottom to colocate them with the other link references?